packages/cli/generators/openapi/schema-helper.js

```diff
@@ -171,11 +171,15 @@ function mapObjectType(schema, options) {
         const itemType =
           propertyType.itemType.kind === 'class'
             ? propertyType.itemType.className
-            : getJSType(propertyType.itemType.name);
+            : getJSType(propertyType.itemType.declaration) ||
```

Comment:
What does the `declaration` here mean?

Reply:
For example, for a type alias `export type IDType = string`:

- name: `IDType
- declaration: `string`